test/lint.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+let assert = require('assert');
+let emu = require('../lib/ecmarkup');
+
+function alg([before, after], marker) {
+  return `<emu-alg>${before}${marker}${after}</emu-alg>`;
+}
+
+let M = '@@@'; // Marks the location of the error
+
+describe('linting', function () {
+  describe('grammar sanity', function () {
+    it('unused parameters', async function () {
+      await assertLint(
+        `
+          <emu-grammar type="definition">
+            Statement[${M}a]: \`;\`
+          </emu-grammar>
+        `,
+        "Parameter 'a' is unused."
```

Comment:
Are these messages being passed through from Grammarkdown's built-in linting? I don't like how its capitalisation and quoting are inconsistent with ours. And I prefer that ours does not change because it's better.

Reply:
Alas, they are.

Reply:
Should we prefix grammarkdown messages or somehow indicate they came from grammarkdown?

Reply:
Ehhh, that doesn't seem like a distinction users are likely to care about.